packages/database/src/database/volunteers.ts

```
@@ -0,0 +1,55 @@
+  if (query?.id) {
+    const result = await prismaClient.volunteers.findUnique({ where: { id: query.id } })
+    return result
+  }
+
+  if (query?.eventid) {
+    where = { eventid: query.eventid }
+  }
+
+  let filter = { where, skip, take, orderBy }
+  if (query?.page || query?.size) {
+    let page = parseInt(query?.page || '0')
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)